storage/tsdb/tsdb_export_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
(nit) 2018

Reply:
We usually set the date to the year that the file was first added to the repo.

Reply:
managed to dig out the comment from @juliusv 	about this.
https://github.com/prometheus/prometheus/pull/4247#discussion_r196587938

I don't mind either way so feel free to open a PR to change it.